src/sdam/server_description.ts

```diff
@@ -18,6 +18,12 @@ const DATA_BEARING_SERVER_TYPES = new Set<ServerType>([
   ServerType.LoadBalancer
 ]);
 
+/** Default in case we are in load balanced mode. */
+const MAX_MESSAGE_SIZE_BYTES = 48000000;
```

Comment:
This is what the 8.0 server returns in its hello responses, so since they are only being used for bulk write I used that. The spec doesn't mention this but without a monitoring connection in load balanced mode we have to default this to something or we cannot batch.

Reply:
Removed.